packages/headless/src/api/platform-client.ts

```diff
@@ -119,6 +120,10 @@ function coveoCloudURL<E extends PlatformEnvironment>(
 export function platformUrl<E extends PlatformEnvironment>(
   options?: URLOptions<E>
 ) {
+  if (options?.orgDomain) {
+    return `https://${options.orgDomain}.org.coveo.com`;
```

Comment:
Why is `.org.coveo.com` hard-coded? Wouldn't developers want to control this part of the url?

Reply:
Is is for multi-region deployments https://docs.coveo.com/en/2976/coveo-solutions/deployment-regions-and-strategies#configuration

Reply:
Got it, thanks. Naming the option something like `multiRegionSubDomain` could be helpful.

Another alternative could be to just accept the full domain (i.e. no option on the platformUrl utility). The reducer can detect whether it is multi-region or not as it is currently, and set the correct analytics endpoint.